cdisc_rules_engine/operations/day_data_validator.py

```diff
@@ -1,16 +1,40 @@
 from cdisc_rules_engine.operations.base_operation import BaseOperation
 from datetime import datetime
+import numpy as np
 
 
 class DayDataValidator(BaseOperation):
     def _execute_operation(self):
-        dtc_value = self.params.dataframe[self.params.target].map(
-            datetime.fromisoformat
-        )
-        rfstdtc_value = self.params.dataframe["RFSTDTC"].map(datetime.fromisoformat)
+        dtc_value = self.params.dataframe[self.params.target].map(self.parse_timestamp)
```

Comment:
Does this require a "domain" parameter be supplied to the operation or will it work with the current domain?

Reply:
If no domain is supplied it will use the current domain.